listenbrainz/spotify_updater/spotify_read_listens.py

```diff
@@ -193,13 +193,14 @@ def make_api_request(user, endpoint, **kwargs):
 def get_user_recently_played(user):
     """ Get tracks from the current user’s recently played tracks.
     """
-    return make_api_request(user, 'me/player/recently-played', limit=50)
+    latest_listened_at_ts = int(user.latest_listened_at.timestamp() * 1000)  # latest listen UNIX ts in ms
+    return  user.get_spotipy_client().current_user_recently_played(limit=50, after=latest_listened_at_ts)
```

Comment:
this bypasses the retry logic that is implemented in `make_api_request`. what we should do is pass the current_user_recently_played function into `make_api_request`

```suggestion
    return  make_api_request(user.get_spotipy_client().current_user_recently_played, limit=50, after=latest_listened_at_ts)
```

something like this

Reply:
done